.github/workflows/release.yml

```
@@ -0,0 +1,45 @@
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Release
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - run: echo "Branch ${{ github.ref }} of repository ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      # Checkout 'fetch-tags' does not work properly, see https://github.com/actions/checkout/issues/701
+      - name: Fecth tags
+        run: git fetch --tags origin
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+      - name: Set up Helm
```

You should be consistent here, use either "setup" or "set up" in both places.

```suggestion
      - name: Setup Go
        uses: actions/setup-go@v5
        with:
          go-version: '1.22'
      - name: Setup Helm
```